File: tests/corpus/positive-fixtures/QA-SE-002/SleepBeforeScreenshot.java
import org.openqa.selenium.TakesScreenshot;
import org.openqa.selenium.WebDriver;

public class SleepBeforeScreenshot {
    public void capture(WebDriver driver) throws Exception {
        driver.get("https://portal.example.test/summary");
        Thread.sleep(4000);
        ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
    }
}

File: tests/corpus/positive-fixtures/QA-SE-002/SleepBeforeScreenshot.cs
using OpenQA.Selenium;

public class SleepBeforeScreenshot
{
    public void Capture(IWebDriver driver)
    {
        driver.Navigate().GoToUrl("https://portal.example.test/summary");
        System.Threading.Thread.Sleep(4000);
        ((ITakesScreenshot)driver).GetScreenshot();
    }
}
